debian/rules

```diff
@@ -21,3 +21,11 @@ override_dh_auto_install:
 	mkdir -p debian/nfd/etc/avahi/services/
 	cp debian/avahi.service debian/nfd/etc/avahi/services/nfd.service
 
+override_dh_installinit:
+	dh_systemd_enable -pnfd --name=nfd nfd.service
+	dh_systemd_enable -pnfd --name=nfd-init nfd-init.service
```

Comment:
You don't need these. `dh_systemd_enable` will already be run before `dh_installinit` since `--with=systemd` is being used.